cinema_service/settings.py

```diff
@@ -11,7 +11,7 @@
 """
 from datetime import timedelta
 from pathlib import Path
-
+from cinema.permissions import IsAdminOrIfAuthenticatedReadOnly
```

Comment:
you don't need to import it. when you add a path in REST_FRAMEWORK, django automatically finds this class